From 415e88c3cfbb34647d0b1bc61e796ce18cce4da6 Mon Sep 17 00:00:00 2001
From: Ismael Luceno <ismael@iodev.co.uk>
Date: Mon, 18 Apr 2022 19:07:40 +0200
Subject: [PATCH 5/5] Remove vpnc-script

An improved replacement is provided by the vpnc-scripts package.

Signed-off-by: Ismael Luceno <ismael@iodev.co.uk>

---
 Makefile |    8 +-------
 1 file changed, 1 insertion(+), 7 deletions(-)

diff --git a/Makefile b/Makefile
index 920bd18b2f25..ca94eeaf01ee 100644
--- a/Makefile
+++ b/Makefile
@@ -66,7 +66,7 @@ ifneq (,$(findstring Apple,$(shell $(CC) --version)))
 CFLAGS += -fstrict-aliasing -freorder-blocks -fsched-interblock
 endif
 
-all : $(BINS) vpnc.8 vpnc-script
+all : $(BINS) vpnc.8
 
 vpnc : $(OBJS) vpnc.o
 	$(CC) -o $@ $^ $(LDFLAGS)
@@ -113,12 +113,6 @@ distclean : clean
 
 install-common: all
 	install -d $(DESTDIR)$(ETCDIR) $(DESTDIR)$(BINDIR) $(DESTDIR)$(SBINDIR) $(DESTDIR)$(MANDIR)/man1 $(DESTDIR)$(MANDIR)/man8 $(DESTDIR)$(DOCDIR)
-	if [ "`uname -s | cut -c-6`" = "CYGWIN" ]; then \
-		install vpnc-script-win $(DESTDIR)$(ETCDIR)/vpnc-script; \
-		install vpnc-script-win.js $(DESTDIR)$(ETCDIR); \
-	else \
-		install vpnc-script $(DESTDIR)$(ETCDIR); \
-	fi
 	install -m600 vpnc.conf $(DESTDIR)$(ETCDIR)/default.conf
 	install -m755 vpnc-disconnect $(DESTDIR)$(SBINDIR)
 	install -m755 pcf2vpnc $(DESTDIR)$(BINDIR)
